symfony/entrypoint.sh

```diff
@@ -1,10 +1,18 @@
 #!/bin/bash
 set -e
 
+: ${WWW_DATA_UID:=`stat -c %u /var/www/html`}
+: ${WWW_DATA_GUID:=`stat -c %g /var/www/html`}
+
 # Change www-data's uid & guid to be the same as directory in host
 # Fix cache problems
-usermod -u `stat -c %u /var/www/html` www-data || true
-groupmod -g `stat -c %g /var/www/html` www-data || true
+if [ "`id -u www-data`" != "$WWW_DATA_UID" ]; then
```

Comment:
I think the test is wrong.
What we need to test is that the $WWW_DATA_UID is not already used by an other user.
If it is the case, we should display a message and don't execute the usermod.

Same for the group.